src/dq_suite/output_transformations.py

```diff
@@ -1,15 +1,22 @@
 import pandas as pd
 
 
+import pandas as pd
+import itertools
+
+# for df_dqValidatie
 def extract_dq_validatie_data(check_name, dq_result):
-    """Function takes a json with the GX output and a string check_name and returns dataframe.
+    """
+    Function takes a json dq_rules,and a string check_name and returns dataframe.
     
     :param df_dq_validatie: A df containing the valid result
     :type df: DataFrame
     :param dq_rules: A JSON string containing the Data Quality rules to be evaluated
     :type dq_rules: str
     :param check_name: Name of the run for reference purposes
     :type check_name: str
+    :param output: A boolean containing is success true or false 
```

Comment:
here we list the input parameters, output is part of the output df and should not be named here